doc/schema.md

```diff
@@ -45,7 +45,7 @@ When the indexer is synced up to the tip of the chain, the hash of the tip is sa
 
 Each funding output (except for provably unspendable ones) results in the following new rows (`H` is for history, `F` is for funding):
 
- * `"H{funding-scripthash}{funding-height}F{funding-txid:vout}{value}" → ""`
+ * `"H{funding-scripthash}{funding-height}F{funding-txid:vout}{value}" → ""` (only saved when output is not provably unspendable or `--index-unspendables` is enabled)
```

Comment:
Maybe add this comment in line 46, something like `(except for provably unspendable ones if `--index-unspendables` is not set)`?